### ISSUE TITLE
Clean up variable scope

### ISSUE THREAD

**Issue by tmcw** [mapbox/vector-tile-query#35]
- https://github.com/mapbox/vector-tile-query/blob/master/index.js#L102

While this will work on practice, it'll throw jshint issues and this sort of code is easily borked into a more serious problem, like an accidental global variable.

``` js
                    if (isNaN(allData[data.hits[i][0].feature_id].attributes()[fields[f]])) {
                        calculateValue = allData[data.hits[i][0].feature_id].attributes()[fields[f]];
                    } else {
                        var distanceRatio = currentPoint[1].distance / (currentPoint[0].distance + currentPoint[1].distance);
                        var queryDifference = (allData[data.hits[i][0].feature_id].attributes()[fields[f]] - allData[data.hits[i][1].feature_id].attributes()[fields[f]]);
                        var calculateValue = allData[data.hits[i][1].feature_id].attributes()[fields[f]] + queryDifference * distanceRatio;
                    }
                    fieldValues.push(calculateValue);
```

See how `calculateValue` only gets a `var` statement on the second time it's mentioned. JavaScript _will_ hoist this `var` as if it was written at the top of the function and scope `calculateValue`, but linters/hinters rarely catch on to this, since it looks like you're temporarily creating a global variable.

Basically: if you initialize a new variable in either part of a if / else, then write a `var variableName;` before the if/else to make sure it'll be locally-scoped.
